### DISCUSSION TITLE
tap/jest for testing

### DISCUSSION THREAD

**Discussion by onosendi** [fastify/fastify#3666] [Q&A] [answered]
Is tap recommended over jest? If so, any particular reason why?

**Comment by zekth** [comment, marked as answer]
Jest overrides globals. Node-tap is the way to go.

**Reply by onosendi** [reply]
Thanks for the quick response.

**Reply by zekth** [reply]
You can find examples of setup througth the `fastify` organization. Most of the plugins are using `node-tap`
